lib/mail/fields/content_type_field.rb

```diff
@@ -144,10 +144,9 @@ def sanatize( val )
 
       # TODO: check if there are cases where whitespace is not a separator
       val = val.
+        gsub(/\s[\w\-]+:\s.*\\r\\n$/, ''). # cases with botched fields like Content-Type: text/html Content-Transfer-Encoding: 8bit\r\n
```

Comment:
We need to strike a balance between "seen in one email we failed to parse" and "things like common issue with emails from Hotmail."

We'd be likely to implement a workaround for common, recurring brokenness, but unlikely to take on the code baggage for rare errors. Particularly when the workaround may cause issues with other valid emails.

In this case, raising an exception and allowing parsing to fail is probably appropriate. Then you can address and fix the problem in your app, likely by correcting the email manually if needed.

We can't promise to parse every email under the sun. But we can make a reasonable best effort :)